### ISSUE TITLE
[NPC] Gatewatcher Iron-Hand in Mechanar HC doesn't cast Jackhammer

### ISSUE THREAD

**Issue by mhdemn** [Atlantiss/NetherwingBugtracker#2942]
[Gatewatcher Iron-Hand](https://db.excalibur.ws/?npc=21526) correctly emotes and players get the warning "swings his hammer menacingly"; but he didn't cast [Jackhammer](https://db.excalibur.ws/?spell=39194) after doing so in any of the two tries we had at it (server crashed during fight, got reset). He did this mechanic correctly in normal difficulty in the last Mechanar I did in phase 2.

**Server Revision**: Noticed in 2339, reposted in 2349 - Assuming it's still bugging.

**Comment by Easyfraggy** [comment]
He does use it - but takes him a long time for first use - initial timer might be too long.

**Comment by Mikik3jr** [comment]
The boss uses this spell BUT somehow the boss interrupts it every time on first cast. Even the Deadly Boss Mod says that I should get the hell out of there, but nothing happens. If the DPS is low you can see that second time it works.

tl;dr: When the boss first time would use the spell he can't.

**Comment by devo6969** [comment]
This is fixed. Engaged the mini boss, damage went off first cast.
PTR -  https://www.youtube.com/watch?v=W5qtp32voaY
